packages/studio-ui/src/components/PropEditor.tsx

```diff
@@ -48,9 +49,18 @@ export default function PropEditor({
     <div className="flex mb-2 text-sm grow flex-col">
       <div className="flex relative">
         {isNested && renderBranchUI()}
-        <label className="pb-1" id={labelTooltipId} htmlFor={inputId}>
-          {propName}
-        </label>
+        <div className="flex flex-row">
+          <label className="pb-1" htmlFor={inputId}>
+            {propName}
+          </label>
+          {tooltip && (
+            <Info
+              id={labelTooltipId}
+              className="ml-3 pb-1"
+              data-testid="prop-tooltip"
```

Comment:
Could we use an aria label instead of a data attribute here?

Reply:
If it's okay, I'd like to stick with the Playwright `testid` for now. Working with the Tooltip, I've come up with some more general WCAG questions for Product. I think those would influence what we want to do ARIA-wise.